Automattic/WP-Job-Manager
# Update tipTip library to use WooCommerce’s version

### Changes proposed in this Pull Request

* The current jQuery tipTip library we use hasn't been updated in over 10 years, and it has some jQuery that has been deprecated like `hover`.   
* WooCommerce also uses this library, and they have updated their library to fix the deprecations, so I'm just taking what they did and moving it here.

### Testing instructions

* Go to the Job Listings page and hover over a few things like "Status", "Featured", "Filled" etc to make sure the hover works.

### Screenshot / Video
![tiptip](https://user-images.githubusercontent.com/3220162/224832825-c8de3a95-5ea2-4033-9894-990b1f770d45.gif)
